eduNEXT/xblock-limesurvey
# feat: add error message field to display errors returned by the API

### Description
This PR adds the `error_message` field that displays the errors returned by the API.
### How to test
1. Configure a survey as in [this PR](https://github.com/eduNEXT/xblock-limesurvey/pull/3).
2. In Studio add or edit the unit with the XBlock, and edit the configuration adding an invalid Access key or Survey ID.
3. From the LMS access to the unit with the XBlock.
4. You should see a error message like this.
![image](https://github.com/eduNEXT/xblock-limesurvey/assets/64033729/215dc323-870d-4bfc-8995-deeca924bf9b)
